### ISSUE TITLE
Add a "copy" button to the read-only view

### ISSUE THREAD

**Issue by Flore-mozfr** [nextcloud/contacts#437]
Hello everyone,
I have shared an address book with other users in my instance, (read-only on purpose), but the problem is : you can’t copy anything (email, phone number, address…). Of course, there’s a icon to compose a mail to the person or to call / message them (not very useful on a computer, imho). And it’s kind of frustrating: you see it, but can’t copy it
So it would be great if there was another icon to just copy the information from the field, so we don’t have to copy it by hand (with all the mistakes). Maybe this icon would only be useful when in read-only mode.

Thanks for this great app and I hope to see this feature in the contacts app, one day

### Expected behaviour
In the contacts webapp (not on anything synced with carddav), one should be able to select any field or at least copy the content of the field.
Maybe an additional icon like the "compose" icon

### Actual behaviour
Can't copy anything in read-only mode.

### Server configuration

**Nextcloud version:** (see Nextcloud admin page)
12.0.3
**Contacts version:** (see Nextcloud apps page)
2.0.1

**Comment by jancborchardt** [comment]
Instead of adding a button, copying should simply be possible. For example by not using read-only input fields, but rather plain text which can be copied.

**Comment by skjnldsv** [comment]
@jancborchardt we went for that before, but decided to roll back because creating too many templates to edit and basically an unorganized structure in the contacts app. :/

**Comment by jancborchardt** [comment]
@skjnldsv right. Is there a different way though to do this which makes copying possible? Like instead of the template switch inserting the "readonly" attribute, it could be used to decide if there's an input field around it at all.

**Comment by skjnldsv** [comment]
We could go for a editable element, but we need to check if it's possible
